ansible/inventory.yml

```diff
@@ -70,3 +70,17 @@
     newrelic_app_name: inventory
   roles:
     - monitoring/newrelic/python-agent-ansible
+
+- name: Smoke test inventory
+  hosts: inventory-web
+  tasks:
+    - name: flush handlers
+      meta: flush_handlers
+    - name: assert app is up
+      uri:
+        url: http://{{ ansible_fqdn }}/
+        status_code: 200
```

Comment:
I get a `302` from https://inventory.data.gov, not a `200`

Reply:
At the service level, yes, but at the host level we are still running on http/80 :( https://github.com/GSA/datagov-deploy/issues/570

Reply:
ah, you mean there's the 302 to max.gov, I'll look into it.

Reply:
Ansible [uri](https://docs.ansible.com/ansible/latest/modules/uri_module.html) follows redirects by default, that's why we see 200 instead of 302.

I'm going to submit another PR to update these checks, since testing the login redirect isn't really that important.

Reply:
Updated the tests in https://github.com/GSA/datagov-deploy/pull/956